chutney/ui/src/app/core/model/scenario/execution.model.ts

```diff
@@ -33,6 +33,7 @@ export class Execution {
     public error?: string,
     public scenarioId?: string,
     public campaignReport?: CampaignExecutionReport,
+    public testCaseTitle?: string
```

Comment:
Can the testCaseTitle be null ? Don't we want to have an error if this happened ?